noteit/src/main/java/com/noteit/exception_handler/CustomErrorResponse.java

```diff
@@ -0,0 +1,70 @@
+package com.noteit.exception_handler;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+import java.util.Date;
+
+@Getter
+@Setter
+
+public class CustomErrorResponse {
+
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
+    private Date timestamp;
+
```

Comment:
In modern Java, we should use `LocalDateTime` from `java.time` package because `java.util.Date` compared to `java.time.LocalDateTime` is lacking a lot of useful features.